pkg/controller/restore-operator/sync.go

```diff
@@ -136,9 +136,12 @@ func (r *Restore) prepareSeed(er *api.EtcdRestore) (err error) {
 	}()
 
 	cs := er.Spec.ClusterSpec
+	if err := cs.Validate(); err != nil {
```

Comment:
This PR should be TLS support only.
Please put validating spec to a different PR/issue.

Reply:
sure